src/ui/QuizUI.js

```diff
@@ -25,9 +25,11 @@ export default class QuizUI {
     this.validationBody = document.getElementById("validation-body");
     this.nextBtn = document.getElementById("next-btn");
     this.finishTemplate = document.getElementById("finish");
+    this.finishedWords = document.getElementById("words-done");
+    this.totalWords = document.getElementById("total-words");
   }
 
-  showItem(item) {
+  showItem(item, wordLeft, totalNumWord) {
```

Comment:
instead of `wordLeft` put `wordsLeft`, and instead of `totalNumWord` put `totalWords`